web/client/observables/persistence/index.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Not clear why this should be an observable. It looks more like an API (provider) to me.

Reply:
Good observation. 
What if we move persistence API (this index.js) object in API folder, leaving here the implementation of `geostore` ? @mbarto @kappu72

Reply:
Not sure, does the developer calling this (the GeoStore impl of this) need to know that this is implemented using observables? I mean, does the caller need to use RxJs or not? Looking at the interface, I cannot understand it.

Reply:
CreateResource and getResource methods return observables, so I guess that RxJs is needed

Reply:
Then I think we should document how the caller should use those methods

Reply:
I agree.
So, summarizing: 
 - [ ] document the interface where missing. updateResource and deleteResource should describe the observable returned (create and get have the `@returns`). 
 - [ ] Move the index.js into api folder
 - [ ] Fix the white space issue ( I introduced it trying to enhance documentation, you can remove it applying these changes)
Did I missed anything else @mbarto  ?

Reply:
No, I think it's everything, thanks